db/seeds.rb

```diff
@@ -22,9 +25,29 @@ def seed_station_database(file_path)
   stations.each do |station|
     city = City.find_by(city: station[:city])
     station.delete_if { |k,v| k == :lat || k == :long || k == :city}
-    city.stations.create(station)
+    city.stations.create!(station)
   end
 end
 
-seed_city_database("db/csv/station.csv")
-seed_station_database("db/csv/station.csv")
+def seed_subscriptions_database(file_path)
+  subscriptions = from_csv(file_path)
+  subscriptions.each do |subscription_type|
+    subscription_type.keep_if {|k,v| k == :subscription_type}
+  end
+  subscriptions.uniq.each {|sub_type| Subscription.create!(sub_type)}
+end
+
+def seed_trips_database(file_path)
+  trips = from_csv(file_path)
+  trips.each do |trip|
+    subscription = Subscription.find_by(subscription_type: trip[:subscription_type])
+    trip.delete_if {|k,v| k == :subscription_type || k == :start_station_name || k == :end_station_name}
```

Comment:
So looking at the database for Station, the station_id (which I assume is the primary key) matches up with the 'id' in the station.csv
<img width="1440" alt="screen shot 2017-07-02 at 8 27 15 am" src="https://user-images.githubusercontent.com/25845955/27770680-45d6a2da-5f00-11e7-8df4-e2b5dbe206ee.png">